Python 2.7.12 (default, Dec  4 2017, 14:50:18) 
[GCC 5.4.0 20160609] on linux2
Type "copyright", "credits" or "license()" for more information.
>>> vars()
{'__builtins__': <module '__builtin__' (built-in)>, '__name__': '__main__', '__doc__': None, '__package__': None}
>>> input ("Cienijamis lietotaj,lūdzu, ievadi vienu skaitli")
Cienijamis lietotaj,lūdzu, ievadi vienu skaitli

Traceback (most recent call last):
  File "<pyshell#1>", line 1, in <module>
    input ("Cienijamis lietotaj,lūdzu, ievadi vienu skaitli")
  File "<string>", line 0
    
    ^
SyntaxError: unexpected EOF while parsing
>>> 10
10
>>> vars()
{'__builtins__': <module '__builtin__' (built-in)>, '__name__': '__main__', '__doc__': None, '__package__': None}
>>> mans_mainigais = input ("Cienijamis lietotaj,lūdzu, ievadi vienu skaitli")
Cienijamis lietotaj,lūdzu, ievadi vienu skaitli10
>>> vars()
{'__builtins__': <module '__builtin__' (built-in)>, 'mans_mainigais': 10, '__package__': None, '__name__': '__main__', '__doc__': None}
>>> mans_mainigais
10
>>> vars(0
     vars()
     
SyntaxError: invalid syntax
>>> vars()
{'__builtins__': <module '__builtin__' (built-in)>, 'mans_mainigais': 10, '__package__': None, '__name__': '__main__', '__doc__': None}
>>> tyoe (mans_mainigais)

Traceback (most recent call last):
  File "<pyshell#10>", line 1, in <module>
    tyoe (mans_mainigais)
NameError: name 'tyoe' is not defined
>>> type (mans_mainigais)
<type 'int'>
>>> vars()
{'__builtins__': <module '__builtin__' (built-in)>, 'mans_mainigais': 10, '__package__': None, '__name__': '__main__', '__doc__': None}
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi skaitli: 

Traceback (most recent call last):
  File "/home/user/RTR105/test_1_20180925.py", line 1, in <module>
    mans_mainigais = input("Cienijamais leitotaj, ludz, ievadi skaitli: ")
  File "<string>", line 0
    
    ^
SyntaxError: unexpected EOF while parsing
>>> vars()
{'__builtins__': <module '__builtin__' (built-in)>, '__file__': '/home/user/RTR105/test_1_20180925.py', '__package__': None, '__name__': '__main__', '__doc__': None}
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi skaitli: 5
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi skaitli: 5
>>> vars(0
     vars()
     
SyntaxError: invalid syntax
>>> vars()
{'__builtins__': <module '__builtin__' (built-in)>, '__file__': '/home/user/RTR105/test_1_20180925.py', 'mans_mainigais': 5, '__package__': None, 'x': 25, '__name__': '__main__', '__doc__': None}
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi skaitli: 89
mans mainigais = 
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi skaitli: 

Traceback (most recent call last):
  File "/home/user/RTR105/test_1_20180925.py", line 1, in <module>
    mans_mainigais = input("Cienijamais leitotaj, ludz, ievadi skaitli: ")
  File "<string>", line 0
    
    ^
SyntaxError: unexpected EOF while parsing
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi skaitli: 186
mans mainigais = 186
>>> vars()
{'__builtins__': <module '__builtin__' (built-in)>, '__file__': '/home/user/RTR105/test_1_20180925.py', 'mans_mainigais': 186, '__package__': None, 'x': 34596, '__name__': '__main__', '__doc__': None}
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi skaitli: 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi skaitli: 45
mans mainigais = 45
vai tu esi ievadijis skaitli: 45
vēl atmiņātagad ir arī mainīgais x = 2025
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi skaitli: 8.6549
mans mainigais =      8.655
rekvizīti,Tu esi ievadijis skaitli:     8.6549
vēl atmiņātagad ir arī mainīgais x =      74.9072940
>>> 
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi skaitli: 9
mans mainigais =      9.000
rekvizīti,Tu esi ievadijis skaitli:     9.0000
vēl atmiņātagad ir arī mainīgais x =      81.0000000
>>> type ()

Traceback (most recent call last):
  File "<pyshell#19>", line 1, in <module>
    type ()
TypeError: type() takes 1 or 3 arguments
>>> type
<type 'type'>
>>> type(x)
<type 'int'>
>>> type (mans_mainigais)
<type 'int'>
>>> type(x)
<type 'int'>
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi skaitli: 8
mans mainigais =      8.000
rekvizīti,Tu esi ievadijis skaitli:     8.0000
vēl atmiņātagad ir arī mainīgais x =      64.0000000
>>> type(mans_mainigais)
<type 'float'>
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi simbolu rindu: 45a

Traceback (most recent call last):
  File "/home/user/RTR105/test_1_20180925.py", line 2, in <module>
    mans_mainigais = 1.* raw_input("Cienijamais leitotaj, ludz, ievadi simbolu rindu: ")
TypeError: can't multiply sequence by non-int of type 'float'
>>> 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi simbolu rindu: 
=============== RESTART: /home/user/RTR105/test_1_20180925.py ===============
Cienijamais leitotaj, ludz, ievadi simbolu rindu: Aaaaaaa

Traceback (most recent call last):
  File "/home/user/RTR105/test_1_20180925.py", line 2, in <module>
    mans_mainigais = 1.* raw_input("Cienijamais leitotaj, ludz, ievadi simbolu rindu: ")
TypeError: can't multiply sequence by non-int of type 'float'
>>> type (mans_mainigais)

Traceback (most recent call last):
  File "<pyshell#25>", line 1, in <module>
    type (mans_mainigais)
NameError: name 'mans_mainigais' is not defined
>>> 

